### DISCUSSION TITLE
Too much variance?

### DISCUSSION THREAD

**Discussion by gkanade** [neherlab/covid19_scenarios#714] [General]
Hi
Just looked at your model today and tried to map parameters that are suitable to my location. I started getting a good sense of the parameters and their impacts and was getting to fit the curve to actual data when I realized some params were having an impact out of proportion. Then I realized that even keeping all params same if I rerun multiple times I get VASTLY different outcomes. I understand there is some randomness built in the model which will likely give difference in outcomes but is there a way to control this ? And have others also observed similar very high variance. Lastly when you say run model 10 or 15 times (higher the number the more stable the results) - which result is ultimately displayed? Because even on this final result that is displayed I see tremendous variation

**Comment by gkanade** [comment]
OK I guess you must be putting out the median case on the display - but then my question is how many runs are recommended to get a good confidence in the model? Seems with 100 runs on the US scenario there is too much variance. In one run I get 67K deaths and next one I get 110K by May 30

**Reply by rneher** [reply]
Our scheme currently samples parameter values from the specified ranges and then plots the sample median (along with 20 and 80th percentile). Since these parameters are exponential growth parameters, the resulting variance can be big. We are discussing ways how to make this less surprising (like plotting a deterministic curve with mean values for parameters). Let us know if you have any suggestions.
